usr/share/rear/finalize/GNU/Linux/250_migrate_disk_devices_layout.sh

```diff
@@ -37,8 +36,5 @@ for file in     [b]oot/{grub.conf,menu.lst,device.map} [e]tc/grub.* [b]oot/grub/
                 fi
         fi
 
-        tmp_layout=$LAYOUT_FILE
-        LAYOUT_FILE="$file"
-        source $SHARE_DIR/layout/prepare/default/320_apply_mappings.sh
-        LAYOUT_FILE=$tmp_layout
+        apply-mappings $file
```

Comment:
Why not quote `$file` as it was quoted before? I think the work of quoting is much less compared to the hassle of analyzing the bug reports of the few users who manage to break ReaR at this place if it is not quoted.